index.md

```diff
@@ -1,49 +1,41 @@
 ---
-layout: default
+layout: home
 title: Home
 section: Overview
 ---
+<p class='alert-big'>Origami is a group of services, modules and tools to make building websites at the FT simpler and faster.</p>
 
-<p class="alert-big">Origami is a set of standards, conventions and tools for building websites at the FT.</p>
+<h2> <i class="fa fa-cubes"></i> Origami modules and services</h2>
 
-<p class="alert-big">Origami aspires to create a <strong>unified style and experience for FT websites</strong>, and <strong>make web development at the FT faster</strong>.</p>
+At the heart of Origami are its services and modules.
 
-This standard is equally for component developers and those who want to build websites using existing components.  These two roles will be distinguished in this standard as **component developer** and **product developer**.
+Origami's modules are bits of HTML, CSS and JavaScript to include in your page. Some of them, like [o-gallery](http://registry.origami.ft.com/components/o-gallery), are fully formed ready to use components. Some of them, like [o-colors](http://registry.origami.ft.com/components/o-colors) or [o-tracking](http://registry.origami.ft.com/components/o-tracking) are tools to incorporate into your own pages and build with.
```

Comment:
"Some of them, like gallery, come with a fully formed look and feel. Some of them, like colors or tracking provide a foundation for building your own interfaces, or provide useful services without any interface at all" <- Yes, this is a much better way of expressing what I was getting at. Thanks.

As for the repetition/use of like/etc, whatever helps you sleep at night is fine with me.